docs/react/guides/migrating-to-react-query-5.md

```diff
@@ -7,6 +7,23 @@ title: Migrating to TanStack Query v5
 
 v5 is a major version, so there are some breaking changes to be aware of:
 
+### Supports a single signature, one object
+
+useQuery and friends used to have many overloads in TypeScript - different ways how the function can be invoked. Not only this was tough to maintain, type wise, it also required a runtime check to see which type the first and the second parameter, to correctly create options. 
+
+now we Only support the object format
+
+```diff
+- useQuery(key, fn, options)
++ useQuery({ queryKey, queryFn, ...options })
+
+- useInfiniteQuery(key, fn, options)
++ useInfiniteQuery({ queryKey, queryFn, ...options })
+
+- useMutation(fn, options)
++ useMutation({ mutationFn, ...options })
+```
```

Comment:
can we list all affected functions here please

Reply:
Done ✅